include/reactor-cpp/time.hh

```diff
@@ -6,30 +6,31 @@
  *   Christian Menard
  */
 
-#pragma once
+#ifndef REACTOR_CPP_TIME_HH
+#define REACTOR_CPP_TIME_HH
 
+#include "logging.hh"
 #include <chrono>
 #include <iostream>
 
+using TimePoint = std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds>;
+using Duration = std::chrono::nanoseconds;
+
 namespace reactor {
 
-using TimePoint = std::chrono::time_point<std::chrono::system_clock,
-                                          std::chrono::nanoseconds>;
+using TimePoint = std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds>;
 using Duration = std::chrono::nanoseconds;
 
-inline TimePoint get_physical_time() {
-  return std::chrono::system_clock::now();
+inline auto get_physical_time() noexcept -> TimePoint { return std::chrono::system_clock::now(); }
 }
 
-inline namespace operators {
-
-std::ostream& operator<<(std::ostream& os, TimePoint tp);
-
-std::ostream& operator<<(std::ostream& os, std::chrono::seconds dur);
-std::ostream& operator<<(std::ostream& os, std::chrono::milliseconds dur);
-std::ostream& operator<<(std::ostream& os, std::chrono::microseconds dur);
-std::ostream& operator<<(std::ostream& os, std::chrono::nanoseconds dur);
+auto operator<<(std::ostream &out_stream, reactor::TimePoint tp) noexcept -> std::ostream &;
+auto operator<<(std::ostream &out_stream, std::chrono::seconds dur) noexcept -> std::ostream &;
+auto operator<<(std::ostream &out_stream, std::chrono::milliseconds dur) noexcept -> std::ostream &;
+auto operator<<(std::ostream &out_stream, std::chrono::microseconds dur) noexcept -> std::ostream &;
+auto operator<<(std::ostream & out_stream, std::chrono::nanoseconds dur) noexcept -> std::ostream &;
 
-}  // namespace operators
+namespace reactor::operators {}
```

Comment:
What happened to the `operators` namespace? Why doesn't it contain the stream operators anymore?